src/Native/System.Native/pal_process.h

```diff
@@ -5,6 +5,9 @@
 
 #include "pal_types.h"
 #include <string.h>
+#include <sys/param.h>
+
+struct rusage_info_v3;
```

Comment:
The v3 is the latest rusage_info struct version number; it gives us more data and the data that we want